graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/runtime/JSContextOptions.java

```diff
@@ -530,19 +530,29 @@ public Map<String, String> apply(String value) {
     public enum UnhandledRejectionsTrackingMode {
         NONE,
         WARN,
-        THROW;
+        THROW,
+        CALLBACK;
 
         @Override
         public String toString() {
             return name().toLowerCase(Locale.ENGLISH);
         }
     }
 
+    public static final String UNHANDLED_REJECTIONS_CALLBACK_NAME = JS_OPTION_PREFIX + "unhandled-rejections-callback";
```

Comment:
have both identifiers defined together because the commends for each reference refers to the name of the other. will fix the spacing though